cmd/service-catalog/template.go

```diff
@@ -14,6 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// This file is copied from k8s.io/kubernetes/pkg/util/template. The file
+// needs to be copied because it is from the 1.8 version of kubernetes. Once
+// the move to kubernetes 1.8 is completed, this file can be removed.
```

Comment:
re: after 1.8
depends whether we want to keep dependency on kubernetes main repo, see https://github.com/kubernetes-incubator/service-catalog/issues/1323